test/inductor/test_codecache.py

```diff
@@ -109,6 +109,7 @@ def reset(self):
 
     @requires_triton()
     @config.patch({"fx_graph_cache": True})
+    @config.patch({"fx_graph_remote_cache": False})
```

Comment:
FYI I think this config.patch thing takes dictionaries, so you could combine with the line above if you happen to like that better; I don't care-- just FYI